web_app/telegram/__main__.py

```diff
@@ -0,0 +1,15 @@
+from . import dp, bot
+from .middleware import DatabaseMiddleware
+import asyncio
+from database.database import SessionLocal
+
+async def main():
```

Comment:
Please, change name to describe what is going on. 
For example: `start_pooling` will be better naming then general `main`